datadog-crashtracker-ffi/src/collector/datatypes.rs

```diff
@@ -68,6 +69,7 @@ pub struct Config<'a> {
     /// If empty, use the default set.
     pub signals: Slice<'a, i32>,
     /// Timeout in milliseconds before the signal handler starts tearing things down to return.
+    /// If 0, uses the default timeout. Otherwise, uses the specified timeout value.
```

Comment:
nit: just so that we know where the default is defined (and what the default is)
```suggestion
    /// If 0, uses the default timeout as specified in
    /// `datadog_crashtracker::shared::constants::DD_CRASHTRACK_DEFAULT_TIMEOUT`. Otherwise, uses
    /// the specified timeout value.
```